internal/pkg/handler/upgrade.go

```diff
@@ -98,6 +98,7 @@ func PerformRollingUpgrade(client kubernetes.Interface, config util.Config, upgr
 				logrus.Errorf("Update for '%s' of type '%s' in namespace '%s' failed with error %v", resourceName, upgradeFuncs.ResourceType, config.Namespace, err)
 			} else {
 				logrus.Infof("Updated '%s' of type '%s' in namespace '%s'", resourceName, upgradeFuncs.ResourceType, config.Namespace)
+				logrus.Infof("Changes detected in '%s' of type '%s' in namespace '%s'", config.ResourceName, config.Type, config.Namespace)
```

Comment:
Changes detected log should come before the Updated log and not after that. It detects the changes and then performs and update.

Reply:
@faizanahmad055 comment has been addressed